app/helpers/greetings_helper.rb

```diff
@@ -0,0 +1,2 @@
+module GreetingsHelper
```

Comment:
Seems that we don't need this helper file, given that we're not making use of it at this moment. We can introduce it if and when it is needed. 

This practice of "deferring the inclusion of something until it's needed" has the advantage that we avoid growing our codebase unnecessarily. Each PR includes only what it needs, allowing for smaller Pull Requests and quicker Code Review cycles.